src/components/content-display/markdown-content/markdown-content.scss

```diff
@@ -76,4 +98,43 @@
     bottom: 0;
     fill: var(--magentaa11y-color-fill-divider-brand);
   }
+
+  textarea, 
+  input[type="text"], 
+  input[type="search"], 
+  input[type="file"], 
+  input[type="email"], 
+  input[type="date"], 
+  input[type="tel"], 
+  input[type="number"], 
+  input[type="password"], 
+  input[type="button"] {
+    border: 2px solid black;
+    border-radius: 2px;
+    box-shadow: 0 0 0 3px #fff;
+    display: block;
+    font-size: 1.125rem;
+    padding: .75rem .5rem;
+    width: auto;
+
+      &:focus-visible {
+        outline: 3px solid #222;
+        transition: .25s ease;
+      }
+  }
+
+  label {
```

Comment:
I think both of these should probably be scoped to the example/.example selector in md-styles

Reply:
and by both, I mean the label styles here and the .hint styles